package/system/opkg/Makefile

```diff
@@ -1,19 +1,19 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
 #
 # Copyright (C) 2006-2015 OpenWrt.org
 # Copyright (C) 2016-2017 LEDE Project
+# Copyright (C) 2018-2021 OpenWrt.org
```

Comment:
```suggestion
# Copyright (C) 2006-2021 OpenWrt.org
```

LEDE no longer exists, long route OpenWrt

Reply:
okay. I deleted LEDE completely. I was not sure how to handle it correctly. BTW, why do only some packages have copy right headers and some don't?